src/constants/misc.js

```diff
@@ -56,7 +57,7 @@ export const LINKS = {
 
 export const STRINGS = {
   CATEGORIES_INCLUDE: 'Categories include:',
-  CONTEXTUAL_DEFINITION: 'Contextual indicators highlight the characteristics of a city that are not amenable to or potentially appropriate for local policy intervention but they can help to explain why a city performs the way it does.',
+  CONTEXTUAL_DEFINITION: 'Context indicators show the characteristics of a city that help us understand what it’s like & why it functions the way it does, based on the measurement framework.',
```

Comment:
Minor: for consistency, '&' should be 'and'.

Also, it's a content thing, but I feel the need to point out: "the measurement framework" - is that supposed to mean something to the user?

Reply:
I doubt it, but so much doesn't make sense.